build.gradle

```diff
@@ -45,10 +45,13 @@ dependencies {
     compile "javax.servlet:javax.servlet-api:3.1.0"
     compile "org.jboss.spec.javax.ws.rs:jboss-jaxrs-api_2.1_spec:1.0.2.Final"
     compile "org.springframework.boot:spring-boot-autoconfigure:$spring_boot_version"
+    compile "org.springframework.boot:spring-boot-starter-actuator:$spring_boot_version"
```

Comment:
If we're adding one of the starters, we should look at our dependency graph and remove duplicates.  I imagine that the starter pulls in `spring-boot-autoconfigure`, `spring-beans`, and `spring-context`.

Personally, I'm torn on whether to use the starters or not.  They do drag in **a lot** of extra dependencies, but they make life easier.

Reply:
Ack. Will take a look at whether I can just pull in a couple deps instead.